File: OrdersMicroservice/Models/Database/OrderItem.cs
namespace OrdersMicroservice.Models.Database
{
    public class OrderItem
    {
        public int Id { get; set; }

        public string ProductId { get; set; } = string.Empty;

        public string StoreId { get; set; } = string.Empty;

        public string Count { get; set; } = string.Empty;

        public string Sum { get; set; } = string.Empty;

        public Order Order { get; set; } = default!;

        public string OrderId { get; set; } = string.Empty;
    }
}

File: client/src/utilities/models/account/OrderItem.ts
import { Item } from "@/utilities/types"

export interface OrderItem {
    productId: string,
    count: string,
    sum: string,
    storeId: string
}

export interface FullOrderItem {
    productId: string,
    count: string,
    sum: string,
    name: Item<string,string>[],
    productUrl: string,
    imageUrl: string
}